graphql/schema/types/scraped-performer.graphql

```diff
@@ -16,11 +18,11 @@ type ScrapedPerformer {
   tattoos: String
   piercings: String
   aliases: String
-  # Should be ScrapedPerformerTag - but would be identical types
-  tags: [ScrapedSceneTag!]
+  tags: [ScrapedTag!]
 
   """This should be a base64 encoded data URL"""
-  image: String
+  image: String @deprecated(reason: "use images instead")
+  images: [String!]
```

Comment:
`ScrapedPerformerData` fragment still uses [`image`](https://github.com/stashapp/stash/blob/991ede83b167763a18ad92d17e6ca04e517c5028/graphql/documents/data/scrapers.graphql#L22), that should be updated to use the non deprecated variant IMO. No point in deprecating something when even the bundled client uses the deprecated field.